### DISCUSSION TITLE
How to change all values in useFieldArray?

### DISCUSSION THREAD

**Discussion by drydenwilliams** [react-hook-form/react-hook-form#7485] [Q&A] [answered]
I would like to have a "Select all" button that updated all the `isChecked` fields in an array: 

https://codesandbox.io/s/react-hook-form-usefieldarray-forked-oddlu?file=/src/index.js:60-73

I can't seem to get it working, any help would be great and really appreciated.

**Comment by Moshyfawn** [comment, marked as answer]
Check out [this codesandbox I've made](https://codesandbox.io/s/react-hook-form-usefieldarray-forked-ydb9g?file=/src/index.js) and see if there's a difference in how you tried to achieve the functionality. _I'm on the road so can't provide details atm, sry_

**Reply by bluebill1049** [reply]
thank you for helping @Moshyfawn
